### DISCUSSION TITLE
How to train model with tags for kie in a properly way

### DISCUSSION THREAD

**Discussion by AbrahamIES** [mindee/doctr#1147] [Q&A]
Hi I was trying to train db_restnet50 to perform detection with labels to perform KIE. I Generate the annotations as in readme in the reference part of the repo. 
First  I tried to train with about 700 annotations and without a pretrain model. But the training performs really bad. recall, precision and iou hardly get 1.00%.

Then I tried to use your pretrained model, removing the last module of the thresh and prob heads of the db_resnet50 arch but the perfomance did not get better, it get worst.

Do have any idea what it could be? I can post more details but I would like to know also if this aproach works for KIE.

Thanks

**Comment by felixdittrich92** [comment]
Ping @aminemindee
